cmake/FindAvro.cmake

```diff
@@ -8,21 +8,31 @@ else()
   find_library (Avro_LIBRARY NAMES avrocpp_s)
 endif()
 
+find_program (Avro_GENCPP NAMES avrogencpp HINTS ${avro_BINARY_DIR})
```

Comment:
> HINTS ${avro_BINARY_DIR}

I would have expected that this part is unnecessary. Did you find it was needed?

Reply:
This was left in as I was trying to debug why avrogencpp wasn't being built/found. Will remove